__tests__/endpoints/getCostumeById.test.js

```diff
@@ -0,0 +1,47 @@
+const {
+    getPool,
+    createTables, 
+    createCostume, 
+} = require('../../db/');
+
+const {
+    getBallroomGown,
+    getButtlessChaps,
+    getBonnet
+} = require('../../db/utils');
+
+const {server} = require('../../app')
+const supertest = require('supertest');
+const request = supertest(server)
+
+
+let pool = getPool();
+
+describe('GET api/costumes/:id', () => {
+    afterAll(async () => {
+        await pool.end()
+    })
+    
+    it('returns costume by ID', async () => {
+        await createTables(pool);
+        await createCostume(pool, getBallroomGown());
+        await createCostume(pool, getButtlessChaps());
+        await createCostume(pool, getBonnet());
+        const response = await request.get('/api/costumes/2');
+        expect(response.status).toBe(200);
+        expect(response.body.name).toBe('buttless chaps');
+      })
+
+      it("should respond with error message if given an ID that does not exist", async () => {
+        expect.hasAssertions();
```

Comment:
?

Reply:
Is this in response to the second test? I am not sure what your question mark is referring to.

Reply:
What is expect.hasAssertions?

Reply:
Ah. Copy pasta from a previous test for an error. It had a try catch statement. I'll remove this.